torchtext/csrc/sentencepiece.cpp

```diff
@@ -1,14 +1,5 @@
 #include <sentencepiece.h>
 
-#ifdef _WIN32
```

Comment:
Why are you deleting this?

Reply:
According to @peterjc123 comments above
> It is okay to remove this registration now because it is not a Python extension anymore. But I'm curious about the crash.

Do you think we still need this?